roles/jd-server/src/lib/mempool/mod.rs

```diff
@@ -7,7 +7,7 @@ use hashbrown::HashMap;
 use roles_logic_sv2::utils::Mutex;
 use rpc_sv2::{mini_rpc_client, mini_rpc_client::RpcError};
 use std::{convert::TryInto, str::FromStr, sync::Arc};
-use stratum_common::{bitcoin, bitcoin::hash_types::Txid};
+use stratum_common::bitcoin::{self, hash_types::Txid};
```

Comment:
why this change?

Reply:
I rebased with dev and it shows you made those commit last month @jbesraa

Reply:
sorry can you point to the commit?

Reply:
this is the commit hash: a76d1022eaca42c9578b5d2bec511acfc2a43e10